### ISSUE TITLE
Issue about evaluation

### ISSUE THREAD

**Issue by RouChan** [Zhuanglong2/T-RODNet#3]
Hi there, 
Thanks for providing a convenient evaluation tool.
However, I found that `gt` sequences you use to evaluate seem to be extracted from training set. I am wondering whether this is valid.

**Comment by Zhuanglong2** [comment]
> Hi there, Thanks for providing a convenient evaluation tool. However, I found that `gt` sequences you use to evaluate seem to be extracted from training set. I am wondering whether this is valid.

Hi RouChan,
Thank you for your interest in our work. The test set for the CRUW dataset does not provide a label, so we have experimentally selected the appropriate test set for convenience. We think this is valid if we are looking at a model performance comparison. Of course, you can also test on the official test set, but that would be cumbersome and not convenient for scientific research.

**Comment by RouChan** [comment]
I got it.
Sincere gratitude to your reply.
